optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/incremental/IncrementalScoreDirectorFactory.java

```diff
@@ -48,6 +49,8 @@ public IncrementalScoreDirectorFactory(Class<? extends IncrementalScoreCalculato
             boolean lookUpEnabled, boolean constraintMatchEnabledPreference) {
         IncrementalScoreCalculator<Solution_> incrementalScoreCalculator = ConfigUtils.newInstance(this,
                 "incrementalScoreCalculatorClass", incrementalScoreCalculatorClass);
+        ConfigUtils.applyCustomProperties(incrementalScoreCalculator, "incrementalScoreCalculatorClass",
```

Comment:
The String literal is repeated one line above; maybe it would be worth introducing a constant.

Reply:
I'd prefer not, it's not going to improve the readability of the code.